cmd/migrate.go

```diff
@@ -64,6 +65,10 @@ func runMigration(_ *cobra.Command, _ []string) error {
 	targetVersion := viper.GetUint(versionFlag)
 	timeout := viper.GetDuration(timeoutFlag)
 
+	if err := run.VerifyDatastoreEngine(uri, run.DatastoreEngine(engine)); err != nil {
```

Comment:
`migrate` package shouldn't have to depend on the `run` package. They are two separate entrypoints. Let's relocate the shared utilities to the cmd utilities package.

Reply:
Refactored shared code to `exec_common` folder